moped-editor/package.json

```diff
@@ -105,5 +105,8 @@
     "eslint-plugin-react": "^7.20.3",
     "eslint-plugin-react-hooks": "^2.5.1",
     "prettier": "^1.19.1"
+  },
+  "engines": {
+    "npm": ">=6.0.0"
   }
-}
+}
```

Comment:
Do you mind if I push up a commit to this branch to add this new line here?

Reply:
sure, please go for it.

Reply:
nvm, let me do it quick, i already have the branch open and I want to go ahead and merge